SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-3761/SpatialTestNetReference/CubeWithReferences.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+
+#include "CubeWithReferences.h"
+#include "Components/StaticMeshComponent.h"
+#include "Materials/Material.h"
+#include "Net/UnrealNetwork.h"
+
+ACubeWithReferences::ACubeWithReferences()
```

Comment:
Just something for the future, and maybe not for this PR: I like the idea of a test actor that you can see and has some desirable basic properties. I think we could create ATestActor parent which is a cube/sphere with the default material, and maybe also replicates. Then other test actors could inherit from it and it would reduce the amount of boilerplate to write.

Reply:
Sure thing, I think making and merging this test actor as soon as possible is preferable, so I will do it in this PR.